### PR TITLE
Ddcnl 1587

### PR DESCRIPTION
**Pull request by thirlz95**

### CODE REVIEW

File: app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html [hunk 1]
```diff
@@ -83,7 +90,11 @@ <h2 class="heading-medium">@messages("nisp.nirecord.voluntarycontributions.h2tit
         <span class="summary">@messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3")</span>
     </summary>
     <div class="panel-indent">
-        <p>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3.message1") )</p>
+        <p>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3.message1"))
+          <a href="@{ApplicationConfig.isleManLink}" tabindex="-1" rel="external" target="_blank" id="nir-external-link" data-journey-click="checkmystatepension:external:iom">
+            @messages("nisp.nirecord.gapsinyourrecord.nationalinsurancecredits.link")
```
Comment (liampan) [inline app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html:95]: is this right? this changes the content doesnt it?

Comment (thirlz95) [reply app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html:95]: fixed now

File: app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html [hunk 2]
```diff
@@ -33,74 +33,79 @@
...
-              <a href="@{ApplicationConfig.moneyAdviceLink}" id="moneyAdviceLink"rel="external" tabindex="-1" target="_blank" data-journey-click="checkmystatepension:external:moneyadvice">@messages("nisp.nirecord.voluntarycontributions.h2title.2.help.link1")</a>
-            </li>
-            <li>
-              <a href="@{ApplicationConfig.pensionWiseLink}" id="pensionWiseLink" rel="external" tabindex="-1" target="_blank" data-journey-click="checkmystatepension:external:pensionwise">@messages("nisp.nirecord.voluntarycontributions.h2title.2.help.link2")</a>
-              &nbsp;@messages("nisp.nirecord.voluntarycontributions.h2title.2.help.link2.message")
-            </li>
+  <h2 class="heading-medium">@messages("nisp.nirecord.voluntarycontributions.h2title.3")</h2>
+  <details class="helptext">
+      <summary class="summary-list-item">
+          <span class="summary">@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle1"))</span>
```
Comment (liampan) [inline app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html:75]: Can we removed the redundant HTML objects

File: app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html [hunk 3]
```diff
@@ -33,74 +33,79 @@
...
+      </summary>
+      <div class="panel-indent">
+          <p>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle1.heading"))</p>
```
Comment (liampan) [inline app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html:78]: *

File: app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html [hunk 4]
```diff
@@ -33,74 +33,79 @@
...
+          <ul class="list-bullet">
+              <li>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle1.message1"))</li>
```
Comment (liampan) [inline app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html:80]: *

File: app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html [hunk 5]
```diff
@@ -33,74 +33,79 @@
...
+              <li>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle1.message2"))</li>
```
Comment (liampan) [inline app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html:81]: *

File: app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html [hunk 6]
```diff
@@ -33,74 +33,79 @@
...
-        <span class="summary">@messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3")</span>
-    </summary>
-    <div class="panel-indent">
-        <p>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3.message1"))
+  <details class="helptext">
+      <summary class="summary-list-item">
+          <span class="summary">@messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3")</span>
+      </summary>
+      <div class="panel-indent">
+          <p id="IOM-message">@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3.message1"))</p>
```
Comment (liampan) [inline app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html:99]: *

File: app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html [hunk 7]
```diff
@@ -33,63 +33,79 @@
...
+              <li>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle1.message2"))</li>
+          </ul>
+      </div>
+  </details>
+  <details class="helptext">
+      <summary class="summary-list-item">
+          <span class="summary">@messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle2")</span>
+      </summary>
+      <div class="panel-indent">
+          <p>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle2.message"))</p>
```
Comment (liampan) [inline app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html:90]: *

**Pull request merged**